examples/streaming/kafka/src/main/scala/org/apache/gearpump/streaming/examples/kafka/wordcount/KafkaWordCount.scala

```diff
@@ -34,27 +34,29 @@ object KafkaWordCount extends App with ArgumentsParser {
   private val LOG: Logger = LogUtil.getLogger(getClass)
 
   override val options: Array[(String, CLIOption[Any])] = Array(
-    "kafka_stream_producer" -> CLIOption[Int]("<hom many kafka producer tasks>", required = false, defaultValue = Some(1)),
+    "source" -> CLIOption[Int]("<hom many kafka producer tasks>", required = false, defaultValue = Some(1)),
     "split" -> CLIOption[Int]("<how many split tasks>", required = false, defaultValue = Some(1)),
     "sum" -> CLIOption[Int]("<how many sum tasks>", required = false, defaultValue = Some(1)),
-    "kafka_stream_processor" -> CLIOption[Int]("<hom many kafka processor tasks", required = false, defaultValue = Some(4))
+    "sink" -> CLIOption[Int]("<hom many kafka processor tasks", required = false, defaultValue = Some(4))
     )
 
   def application(config: ParseResult) : StreamApplication = {
-    val kafkaStreamProducerNum = config.getInt("kafka_stream_producer")
+    val sourceNum = config.getInt("source")
     val splitNum = config.getInt("split")
     val sumNum = config.getInt("sum")
-    val kafkaStreamProcessorNum = config.getInt("kafka_stream_processor")
+    val sinkNum = config.getInt("sink")
 
-    val kafkaConfig = KafkaConfig(ConfigFactory.parseResources("kafka.conf"))
     val appConfig = UserConfig.empty
-      .withValue(KafkaConfig.NAME, kafkaConfig)
+      .withInt(DataSourceConfig.SOURCE_READ_BATCH_SIZE, 1000)
```

Comment:
Can we add some default value for this? If user doesn't set it, by default it is 1000 (e.g.).